rustfmt.toml

```diff
@@ -0,0 +1 @@
+disable_all_formatting = true
```

Comment:
Its a good idea to disable rustfmt if you don't use it.
edit This requires nightly rustfmt, so I've just enabled it. The changes weren't that intrusive.

By adding a rustfmt.toml to the repository with disable_all_formatting = true (or ignore = ["."]) people who work on the branch locally won't have their rustfmt do sweeping changes.

I would recommend using rustfmt though, so removing that line and adding the simple edition = "2018" is a nice upgrade.